src/lando/api/tests/test_hooks.py

```diff
@@ -10,7 +10,7 @@
 )
 
 
-def test_app_wide_headers_set(client):
+def test_app_wide_headers_set(db, client):
```

Comment:
Where does `db` come from? Is it a fixture? I can't find its definition (not that it impacts the tests).

Reply:
Good point. This is actually a fixture [provided](https://pytest-django.readthedocs.io/en/latest/helpers.html#db) by pytest-django, though after going over the documentation again, it seems it is intended to be used in other fixtures, and not as a shortcut/alternative to marking the test with `django_db`.

Though it works as expected, I think it's better to explicitly mark those tests as needing the db. I'll file a bug to fix other occurrences in the future, but will change these in this PR.